Content.Client/Store/Ui/StoreBoundUserInterface.cs

```diff
@@ -83,4 +91,19 @@ protected override void Dispose(bool disposing)
         _menu?.Close();
         _menu?.Dispose();
     }
+
+    private void UpdateListingsWithSearchFilter()
+    {
+        if (_menu == null)
+            return;
+
+        var filteredListings = new HashSet<ListingData>(_listings);
+        if (!string.IsNullOrEmpty(_search))
+        {
+            filteredListings.RemoveWhere(listingData => !Loc.GetString(listingData.Name).Trim().ToLowerInvariant().Contains(_search) &&
```

Comment:
i'm reasonably sure that this name can be null, in which case it'll use the entity name. You'll probably need to make a helper method since this is a common issue with this system IIRC.

Reply:
Thanks for checking on it! I'll write up a fix next time I'm at my machine.